tsdb/head_wal.go

```diff
@@ -820,7 +821,7 @@ func decodeSeriesFromChunkSnapshot(d *record.Decoder, b []byte) (csr chunkSnapsh
 	// TODO: figure out why DecodeLabels calls Sort(), and perhaps remove it.
 	csr.lset = d.DecodeLabels(&dec)
 
-	csr.chunkRange = dec.Be64int64()
+	_ = dec.Be64int64() // backwards-compatibility: now unused
```

Comment:
is it?
```suggestion
	_ = dec.Be64int64() // Backwards-compatibility; now unused.
```

Reply:
If I put "was chunkRange but now unused" does that make it clearer?

Reply:
Yes it does